src/System.Private.Xml/src/System/Xml/Core/HtmlUtf8RawTextWriter.cs

```diff
@@ -711,7 +711,7 @@ private unsafe void WriteUriAttributeText(char* pSrc, char* pSrcEnd)
                             break;
                         default:
                             const string hexDigits = "0123456789ABCDEF";
-                            fixed (byte* pUriEscapingBuffer = _uriEscapingBuffer)
+                            fixed (byte* pUriEscapingBuffer = &_uriEscapingBuffer[0])
```

Comment:
Yes, the array is allocated in the [`Init` method](https://github.com/dotnet/corefx/blob/2b6aea47dfe63146cadacdf9e9d7b14825c366c1/src/System.Private.Xml/src/System/Xml/Core/HtmlEncodedRawTextWriter.cs#L465), which is [called by both constructors](https://github.com/dotnet/corefx/blob/2b6aea47dfe63146cadacdf9e9d7b14825c366c1/src/System.Private.Xml/src/System/Xml/Core/HtmlEncodedRawTextWriter.cs#L43-L52).

I went ahead and added an assert above this `fixed` statement, just to be clear that we're expecting it not to be null or empty.